index.js

```diff
@@ -1,21 +1,42 @@
+/*jshint node: true */
+'use strict';
+
 var path = require('path');
+var mergeTrees = require('broccoli-merge-trees');
+var Funnel = require('broccoli-funnel');
 
 module.exports = {
   name: 'ember-data-factory-guy',
 
+  treeForVendor: function() {
```

Comment:
Is this making a vendor file out of the npm file?
Can you look at issue:  https://github.com/danielspaniel/ember-data-factory-guy/issues/206
 and see if you can hook that up somehow? 
 I am trying to take the tests/factories directory and include those files in the development build ( don't really care where .. as in vendor.js ,  or the app ( appName.js ) or even a separate js file that I can then include the way you did here
 ( after you made the file ) .. as long as those factory files are watched and updated when changed I would be your best friend for a day if you could make that work.